src/main/java/lotto/view/InputView.java

```diff
@@ -6,37 +6,77 @@
 import java.util.List;
 import java.util.Scanner;
 
-import static lotto.service.LottoGame.checkWinningNumbers;
+import static lotto.data.LottoUtil.validateBonusNumber;
+import static lotto.data.LottoUtil.validateInputLottoNumber;
 
 public class InputView {
     private static final Scanner scanner = new Scanner(System.in);
     private static final String QUESTION_PURCHASE_AMOUNT = "구입금액을 입력해 주세요.";
+    private static final String QUESTION_MANUAL_LOTTO_COUNTS = "수동으로 구매할 로또 수를 입력해 주세요.";
+    private static final String QUESTION_MANUAL_LOTTO_NUMBERS = "수동으로 구매할 번호를 입력해 주세요.";
     private static final String QUESTION_WINNING_NUMBERS = "지난 주 당첨 번호를 입력해 주세요.";
+    private static final String QUESTION_BONUS_NUMBERS = "보너스 볼을 입력해 주세요.";
     private static final String DELIMITER = ",";
 
-    private InputView(){
+    private InputView() {
         //
     }
 
     public static int getPurchaseAmount() {
         System.out.println(QUESTION_PURCHASE_AMOUNT);
         int price = scanner.nextInt();
+        System.out.println();
+        return price;
+    }
+
+    public static int getManualLottoCounts() {
+        System.out.println(QUESTION_MANUAL_LOTTO_COUNTS);
+        int price = scanner.nextInt();
```

Comment:
```suggestion
        int 수동구매갯수 = scanner.nextInt();
```

`수동으로 구매할 로또 수를 입력해 주세요.` 와 `price` 이름과 어울리지 않는것 같아요! 🤔